docs/api/conventions.md

```diff
@@ -187,8 +187,8 @@ Please note that you can use either `.jsx` or `.tsx` file extensions depending o
 <!-- prettier-ignore -->
 ```markdown [3]
 app/
-├── routes/
-└── root.tsx
```

Comment:
Does this look bad on non-macs? I think these should be fine as-is... It's just the table thing we had that should be changed I think.

Reply:
Windows uses the same characters, so should be fine.
```
Volume serial number is EE88-C117
C:\PROJECTS\SCRATCH\MY-REMIX-APP\APP
│   entry.client.tsx
│   entry.server.tsx
│   root.tsx
│
└───routes
        index.tsx
```